.github/workflows/master.yml

```diff
@@ -27,7 +27,50 @@ jobs:
         export PATH="$HOME/miniconda/bin:$PATH"
         source activate test-environment
         ./lint.sh
+  r:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - uses: r-lib/actions/setup-r@v1
+    - name: Query dependencies
+      run: |
+        install.packages('remotes')
+        saveRDS(remotes::dev_package_deps("mlflow/R/mlflow", dependencies = TRUE), ".github/depends.Rds", version = 2)
+        writeLines(sprintf("R-%i.%i", getRversion()$major, getRversion()$minor), ".github/R-version")
+      shell: Rscript {0}
 
+    - name: Cache R packages
+      if: runner.os != 'Windows'
+      uses: actions/cache@v1
```

Comment:
Looks like the latest version of the `cache` action is v2: https://github.com/actions/cache#whats-new

Out of curiosity, any reason we're using v1 here? If not, maybe we could just upgrade to v2 preemptively

Reply:
right. TBH, I just copied from the template from r-lib/actions. Let's try `v2`.